docs/dialogflow-platforms.rst

```diff
@@ -527,6 +527,185 @@ Or you can also handle these values from your views file
   },
 
 
+- Button Template
+You need to include in your controller the following field: ``facebookButtonTemplate``, which takes an object with an array of buttons to be taken as items in a list of buttons. For more information about the button template, check how to add a `Button Template <https://developers.facebook.com/docs/messenger-platform/send-messages/template/button>`_.
+
+.. code-block:: javascript
+  const {
+    FacebookButtonTemplateBuilder,
+    FacebookTemplateBuilder,
+  } = require('voxa');
+
+  app.onState('someState', () => {
+    const buttonBuilder1 = new FacebookButtonTemplateBuilder();
+    const buttonBuilder2 = new FacebookButtonTemplateBuilder();
+    const buttonBuilder3 = new FacebookButtonTemplateBuilder();
+    const facebookTemplateBuilder = new FacebookTemplateBuilder();
+
+    buttonBuilder1
+      .setPayload("payload")
+      .setTitle("View More")
+      .setType("postback");
```

Comment:
I think for the `type` property you should create an enum in Voxa's code. I can see there are 9 [types of buttons](https://developers.facebook.com/docs/messenger-platform/send-messages/buttons) users can use and they better go to Voxa docs instead of Facebook docs to see which button type they can use.